src/ApiService/Tests/Tests.csproj

```diff
@@ -2,8 +2,6 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
-    <Nullable>enable</Nullable>
```

Comment:
this is supposed to be a compiler flag to help identify potential null issues
is it removed because it's not helping ?

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/attributes/nullable-analysis

Reply:
I know i just removed it only for the tests project because is it was causing a lot of warning.  So i removed it instead of having a lot of null check that would make the tests less readable.

Reply:
brought it back https://github.com/microsoft/onefuzz/pull/1753